app/store.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, applyMiddleware } from 'redux';
+import { createLogger } from 'redux-logger';
+import thunkMiddleware from 'redux-thunk';
+import axios from 'axios';
+
```

Comment:
maybe we can import some redux dev tools for easier redux debugging such as: 

createLogger from 'redux-logger';
composeWithDevTools from 'redux-devtools-extension';